docker-compose.yml

```
@@ -8,6 +8,7 @@ services:
       - "80:80"
     volumes:
       - ./:/silauth
+      - ./development/run-debug.sh:/data/run.sh
```

I wouldn't turn on debugging by default. What about adding this line but leaving it commented out?
```suggestion
      # - ./development/run-debug.sh:/data/run.sh
```

---

Right, I think I intended to remove that. Thanks.